editor/import/SCsub

```diff
@@ -3,3 +3,6 @@
 Import("env")
 
 env.add_source_files(env.editor_sources, "*.cpp")
+
+if not env["disable_3d"]:
+    SConscript("3d/SCsub")
```

Comment:
Did you make sure that none of these headers is ever included in another file that's compiled even when `disable_3d` is enabled? Because that would mean `disable_3d` would fail linking.

Reply:
Actually I see such issues in this very PR, since the header includes should all be enclosed in `#ifndef _3D_DISABLED` for this to work.

But the reason this might have worked in your testing is that `disable_3d` is not a supported option for the editor, so this check is always going to be true. So I would remove it as it's a false promise of a possibility which isn't supported.

Reply:
Correct, this is here as a hopeful bit of future-proofing, but as you've explained to me before in other PRs, it's not a good idea. I've pushed a change to remove this, and to remove the nearly-empty SCsub in favor of including `3d/*.cpp` which could always be changed later if another SCsub is really needed.